src/test/java/wooteco/subway/service/PathServiceTest.java

```diff
@@ -42,68 +42,84 @@ class PathServiceTest {
-        List<Station> stations = new ArrayList<>();
-        for (char c = 'a';  c <= 'k' ; c++) {
-            Station station = new Station(String.valueOf(c));
-            stations.add(stationDao.save(station));
+    private void createStations() {
+        List<Station> stations = List.of(STATION1, STATION2, STATION3, STATION4, STATION5, STATION6, STATION7, STATION8, STATION9, STATION10, STATION11);
+        for (Station station : stations) {
+            Station save = stationDao.save(station);
+            System.out.println(save.getId());
         }
-        return stations;
     }
 
     @Test
     @DisplayName("최단거리 경로를 반환한다.")
-    void findShortestPath(){
-        PathResponse pathResponse = pathService.findShortestPath(stations.get(0).getId(), stations.get(8).getId());
+    void findShortestPath() {
+        PathRequest pathRequest = new PathRequest(STATION1.getId(), STATION9.getId(), 20);
+        PathResponse pathResponse = pathService.findShortestPath(pathRequest);
         assertThat(pathResponse.getStations()).containsExactly(
-            StationResponse.from(stations.get(0)),
-            StationResponse.from(stations.get(1)),
-            StationResponse.from(stations.get(2)),
-            StationResponse.from(stations.get(7)),
-            StationResponse.from(stations.get(8))
+                StationResponse.from(STATION1),
+                StationResponse.from(STATION2),
+                StationResponse.from(STATION3),
+                StationResponse.from(STATION8),
+                StationResponse.from(STATION9)
         );
         assertThat(pathResponse.getDistance()).isEqualTo(58);
-        assertThat(pathResponse.getFare()).isEqualTo(2150);
+        assertThat(pathResponse.getFare()).isEqualTo(3050);
     }
 
     @Test
     @DisplayName("출발역과 도착역이 같으면 예외를 던져야 한다.")
     void findSameStationsPath() {
-        assertThatThrownBy(() -> pathService.findShortestPath(stations.get(0).getId(), stations.get(0).getId()))
-            .hasMessage("출발역과 도착역이 동일합니다.")
-            .isInstanceOf(IllegalArgumentException.class);
+        PathRequest pathRequest = new PathRequest(STATION1.getId(), STATION1.getId(), 0);
+        assertThatThrownBy(() -> pathService.findShortestPath(pathRequest))
+                .hasMessage("출발역과 도착역이 동일합니다.")
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @Test
+    @DisplayName("노선 추가 요금을 더해서 반환한다.")
+    void findShortestPath_lineFare() {
+        PathRequest pathRequest = new PathRequest(STATION1.getId(), STATION5.getId(), 20);
+        PathResponse pathResponse = pathService.findShortestPath(pathRequest);
+        assertThat(pathResponse.getStations()).containsExactly(
+                StationResponse.from(STATION1),
+                StationResponse.from(STATION2),
+                StationResponse.from(STATION5)
+        );
+        assertThat(pathResponse.getDistance()).isEqualTo(9);
+        assertThat(pathResponse.getFare()).isEqualTo(1750);
```

Comment:
```suggestion
    void findShortestPath_lineFare() {
        PathRequest pathRequest = new PathRequest(STATION1.getId(), STATION5.getId(), 20);
        
        PathResponse pathResponse = pathService.findShortestPath(pathRequest);
        
        assertThat(pathResponse.getStations()).containsExactly(
                StationResponse.from(STATION1),
                StationResponse.from(STATION2),
                StationResponse.from(STATION5)
        );
        assertThat(pathResponse.getDistance()).isEqualTo(9);
        assertThat(pathResponse.getFare()).isEqualTo(1750);
```

비즈니스 흐름별로 개행을하면 테스트코드에 가독성을 챙길 수 있을 것 같아요.